### DISCUSSION TITLE
time course image data structure

### DISCUSSION THREAD

**Discussion by ewabb** [SchmollerLab/Cell_ACDC#176] [Q&A] [answered]
I'm not sure how to save my .tif files given that I have multiple channels, positions, and a time course (all in single-channel individual .tif files). Even when I only work with the files from a single position, the 'Create data structure from microscopy files' does not allow for a time course. It either tries to force multiple sites or gives an error when I set up number of timepoints correctly. Can the ACDC work correctly with a time course in multiple channels?

**Comment by ElpadoCan** [comment, marked as answer]
Discussion is continuing on issue #175
